packages/bot-web-ui/src/app/app.jsx

```diff
@@ -65,17 +58,8 @@ const App = ({ passthrough }) => {
         setIsLoading(true);
         active_symbols.retrieveActiveSymbols(true).then(() => {
             setIsLoading(false);
-            if (!show_dashboard) {
-                onMount();
-            }
         });
-        return () => {
-            if (!show_dashboard) {
-                onUnmount();
-            }
-        };
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [onMount, onUnmount]);
+    }, []);
```

Comment:
This will throw an eslint warning because of missing dependencies.

Reply:
`added eslint-disable-next-line  react-hooks/exhaustive-deps` to ignore the warning ✅